lib/index.js

```diff
@@ -19,6 +72,10 @@ function onlyFiles(schemas) {
 function getAllObjects(baseQuery)  {
   var allObjects = [];
   var next = function(startIndex) {
+    if (startIndex > 10000) {
```

Comment:
Uhh very good call. 
Instead should we order by createdAt and use the last object all the times? 1 logic instead of 2?